app/views/admin/merchants/index.html.erb

```diff
@@ -1,14 +1,14 @@
 <h1>All Merchants</h1>
 <section>
-  <% @merchants.each do |merchant| %>
+  <% @merchants.map do |merchant| %>
```

Comment:
Probably should change this back to an each...

Reply:
I agree -- this is cute Ruby -- but if we need more links and info added to each `merchant` this makes that difficult